src/main.sh

```diff
@@ -241,9 +249,19 @@ function main {
       ;;
     *)
       echo "Error: Must provide a valid value for terragrunt_subcommand"
-      exit 1
+      mainExitCode=1
       ;;
   esac
+
+  # Process the working dir if we have a non-errored exit code
+  if [ ${mainExitCode} -ne 1 ]; then
+    # Pass the directory used for processing terraform to the outputs
+    terraformDir=$(findTerraformDir)
```

Comment:
**Suggestion:** Wrap this in quotes so that if multiple values are returned, or if there are spaces in the path, it will still work.
```suggestion
    terraformDir="$(findTerraformDir)"
```